astropy/utils/iers/tests/test_leap_second.py

```diff
@@ -181,13 +186,17 @@ def test_erfa_found(self):
             ls = iers.LeapSeconds.open()
         assert ls.meta['data_url'] == 'erfa'
 
-    def test_builtin_found(self):
+    def test_builtin_found_and_not_expired(self):
         # Set huge maximum age such that built-in file is always OK.
         # If we remove 'erfa', it should thus be found.
         self.remove_auto_open_files('erfa')
         with iers.conf.set_temp('auto_max_age', 100000):
             ls = iers.LeapSeconds.open()
         assert ls.meta['data_url'] == iers.IERS_LEAP_SECOND_FILE
+        # The built-in file should be kept up to date.
+        # TODO: would be nice to have automatic PRs for this!
```

Comment:
Is this request the same as the one at #1697 ?

Reply:
Yes, but now every half year we'll get a reminder... (more seriously, will add this to #1697)

Reply:
and I see that issue ends with me promising to open a PR to the release procedure... :man_facepalming:

Reply:
> release procedure

Not sure how I feel about it being part of the release procedure. However it certainly can be a "last before release" type of reminder, similarly to the bundled libraries. See https://github.com/astropy/astropy/issues/9018

Reply:
maybe we can make this a cron only test (or simply just flag with remote_data thus allowed to fail on PRs), so as not to introduce false positives while the update PR is not merged?

Reply:
:+1: to `remote_data` with a comment to explain why. But it will have to be a follow-up PR.

Reply:
OK, I'll make a quick PR that marks this remote_data - agreed that we don't really want to have regular PRs start failing with it!